libraries/ui-library/src/components/six-input/six-input.tsx

```diff
@@ -205,45 +205,52 @@ export class SixInput {
     this.handleClearClick = this.handleClearClick.bind(this);
     this.handlePasswordToggle = this.handlePasswordToggle.bind(this);
     this.handleSlotChange = this.handleSlotChange.bind(this);
-
-    this.host.shadowRoot.addEventListener('slotchange', this.handleSlotChange);
+    this.host.shadowRoot?.addEventListener('slotchange', this.handleSlotChange);
   }
 
   componentWillLoad() {
-    this.defaultValue = this.value;
+    this.defaultValue = this.getValue();
     this.handleSlotChange();
   }
 
   componentDidLoad() {
-    this.eventListeners.add(this.input, 'invalid', (event) => {
+    const nativeInput = this.nativeInput;
+    if (nativeInput === undefined) {
```

Comment:
Purely a matter of taste, but I prefer the convention to use `!= null` / `== null` to check for null / undefined if not intentionally checked for just either of them and to avoid using undefined wherever possible.